### PR TITLE
1022-Update adopter dashboard

### PR DESCRIPTION
**Pull request by wandergithub**
# 🔗 Issue

https://github.com/rubyforgood/homeward-tails/issues/1022
# ✍️ Description

Update adoper dashboard
# 📷 Screenshots/Demos
image

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -648,6 +648,8 @@ en:
           pets: Pets
           applications: Applications
           rescues: Rescues
+          header: "Welcome to Beja Pet Rescue!"
+          body_text: "At Baja Pet Rescue, we’re dedicated to rescuing, rehabilitating, and rehoming animals in need. Your support helps us provide care, love, and hope to these pets until they find their forever homes."
```
Comment (jmilljr24) [inline config/locales/en.yml:652]:
```suggestion
          header: "Welcome to %{organization}!"
          body_text: "At %{organization}, we’re dedicated to rescuing, rehabilitating, and rehoming animals in need. Your support helps us provide care, love, and hope to these pets until they find their forever homes."
```

File: app/views/organizations/adopter_fosterer/dashboard/index.html.erb [hunk 2]
```diff
@@ -1,36 +1,24 @@
...
-          <% card.with_title { t(".rescues") } %>
-          <% card.with_icon_name { "star" } %>
-          <% card.with_value { "TBD" } %>
-        <% end %>
+    <div class="justify-content-md-between">
+      <div class="row d-flex">
+        <div class=" col-xxl-5 col-xl-6 col-lg-6 col-12">
+          <div>
+            <h1 class="fw-bold"><%= t('.header') %></h1>
+            <p class="fs-4 mb-4"><%= t('.body_text') %></p>
```
Comment (jmilljr24) [inline app/views/organizations/adopter_fosterer/dashboard/index.html.erb:8]: Can we pass in the `Current.organization.name `?

File: app/assets/images/dashboard.jpeg
Comment (kasugaijin) [inline app/assets/images/dashboard.jpeg:1]: This is a BIG file! Can you please compress with https://tinypng.com/
This should be as small as possible, like 100kb or less, ideally.

File: config/locales/en.yml (hunk 1 above)
Comment (kasugaijin) [reply config/locales/en.yml:652]: This ^ these should be dynamically generated

**Comment by wandergithub** [comment]
image

**Review by kasugaijin — APPROVED** [review]
@wandergithub thank you!

**Comment by wandergithub** [comment]
@kasugaijin Once any issue is available with some functionality, testing, and less front-end related let me know.

**Pull request merged**